ndbench-core/src/main/java/com/netflix/ndbench/core/NdBenchDriver.java

```diff
@@ -103,19 +101,16 @@
         this.settableConfig = settableConfig;
         this.rpsCount = new RPSCount(readsStarted, writesStarted, readLimiter, writeLimiter, config, ndBenchMonitor);
 
-        Runtime.getRuntime().addShutdownHook(new Thread() {
-            @Override
-            public void run() {
-                logger.info("*** shutting down NdBench server since JVM is shutting down");
-                NdBenchDriver.this.stop();
-                try {
-                    NdBenchDriver.this.shutdownClient();
-                    Thread.sleep(2000);
-                } catch (Exception e) {
-                    //ignore
-                }
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            logger.info("*** shutting down NdBench server since JVM is shutting down");
+            NdBenchDriver.this.stop();
+            try {
+                NdBenchDriver.this.shutdownClient();
+                Thread.sleep(2000);
```

Comment:
if the JVM is shutting down, do we have enough time to sleep here?

Reply:
This is the best effort, does not need to wait